llama.cpp

```diff
@@ -1708,6 +1710,7 @@ enum e_model {
     MODEL_4B,
     MODEL_7B,
     MODEL_8B,
+    MODEL_12B,
```

Comment:
I think the mapping to the string `"12B"` is missing. Search for references of the other nearby model sizes to find where that should be done.